packages/specification/eps-prescription-status-update-api.yaml

```diff
@@ -53,15 +55,37 @@ info:
     For more details see [Network access for APIs](https://digital.nhs.uk/developer/guides-and-documentation/network-access-for-apis).
 
     ## Security and Authorisation
-    This API only supports application-restricted access, meaning we authenticate the calling application but not the end user.
+    This API only supports [application-restricted access](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
+    ### Application-restricted Access
     The end user could be:
-    a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
-    not present at all - for example as part of a back end process integrated with a shipping system
+
+    * a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
+    * a patient - in which case you must ensure they are authenticated locally
+    * not present at all - for example as part of a back end process to check NHS numbers for data flowing from one system to another
+
 
     To use this access mode, use the following security pattern:
     * [Application-restricted RESTful API - signed JWT authentication](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-signed-jwt-authentication)
 
+    ### Errors
+    We use standard HTTP status codes to show whether an API request succeeded or not.
+    They are usually in the range:
+
+    * 200 to 299 if it succeeded, including code 202 if it was accepted by an API that needs to wait for further action
+    * 400 to 499 if it failed because of a client error by your application
+    * 500 to 599 if it failed because of an error on our server
+
+    Errors specific to each API are shown in the Endpoints section, under Response. See our [reference guide](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#http-status-codes) for more on errors.
+
+    ## Open source
+    You might find the following [open source](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#open-source) resources useful:
+
+    | Resource | Description | Links |
+    |----------|-------------|-------|
```

Comment:
we should have links to this repo and the FHIR validator lambda repo here.

Reply:
5df2893b05761db3c7e45188592ee1fcae43a819